guides/common/modules/proc_flatpak-remote-for-smartproxy-setup.adoc

```diff
@@ -0,0 +1,27 @@
+[id="Flatpak_Remote_for_Smartproxy_{context}"]
+= Flatpak remote for {SmartProxy} setup
+
+This procedure configures {SmartProxy} servers to synchronize and distribute Flatpak repositories to managed hosts. 
+
+.Prerequisites
+* Ensure the Flatpak repositories are synced to the {SmartProxy} as they are for any other repository.
+
+.Procedure
+. Run the following command on the managed host to add the {SmartProxy} as a Flatpak remote:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ flatpak --user remote-add --authenticator-name=org.flatpak.Authenticator.Oci katello oci+https://capsule.example.com/pulpcore_registry/
+----
+. Log in using podman:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ podman login
+----
+. Install the Firefox Flatpak:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ flatpak install firefox
+----
```

Comment:
Add a note, this may require additional login to registry if they haven't saved credentials.
This is the same as Step 3 of https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/8/html/using_the_desktop_environment_in_rhel_8/assembly_installing-applications-using-flatpak_using-the-desktop-environment-in-rhel-8#proc_enabling-the-red-hat-flatpak-remote_assembly_installing-applications-using-flatpak